CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v0.2.5
+
+### [0.2.5](https://github.com/openfga/js-sdk/compare/v0.2.4...v0.2.5) (2023-04-21)
+
+- feat(client): implement `listRelations` to check what relationships a user has with an object
+- feat(client): proper parallel limit for batch fns (BatchCheck, etc..)
+- chore(ci): publish provenance data
+- chore(deps): update dependencies
+
```

Comment:
Do we also want to call out that we expect schema version be returned (i.e., it will not work with old version of OpenFGA)?

Reply:
Will add it